src/CRUD.php

```diff
@@ -89,7 +90,7 @@ public function setModel(\atk4\data\Model $m, $fields = null)
             if ($executor instanceof View) {
                 $executor->stickyGet($this->name.'_sort', $this->getSortBy());
             }
-            $single_record_action->fields = ($executor instanceof jsUserAction) ? false : $this->editFields ?? true;
+            $single_record_action->fields = ($executor instanceof jsUserAction) ? false : ($this->editFields ?? []);
```

Comment:
@ibelar - can you plese check this?

Reply:
Needs to be set to true not empty array for UserAction to properly set action->fields in form.